src/gadget_bound_check.rs

```diff
@@ -114,6 +115,11 @@ pub fn verify_proof_of_bounded_num(lower: u64, upper: u64, max_bits_in_val: usiz
     verifier.verify(&proof, &pc_gens, &bp_gens)
 }
 
+fn count_log_bits(number: u64) -> usize {
```

Comment:
I know that this is test-code so my feedback doesn't matter that much but:
1. `count_log_bits` is not an appropriate name since you really counting the bits needed to represent `number` and not a logarithm of the number of bits.
2. Secondly, you can write the logic in a simpler way as `64 - number.leading_zeros()`. `leading_zeros` will essentially return the number of unused bits and you can subtract that from 64 to get the number of used bits.